src/lib.rs

```diff
@@ -193,10 +163,10 @@ pub async fn start(
                                         results.push(streamer_msg);
                                     },
                                     Ok(Err(e)) => {
-                                        tracing::error!(target: BLOCKSAPI_RS, "Error converting message: {}", e);
+                                        tracing::error!(target: BLOCKSAPI, "Error converting message: {}", e);
```

Comment:
Please decide for once do you use target or not, because in some other files you don't